sql/core/src/test/scala/org/apache/spark/sql/UDFSuite.scala

```diff
@@ -247,4 +248,21 @@ class UDFSuite extends QueryTest with SharedSQLContext {
       sql("SELECT tmp.t.* FROM (SELECT testDataFunc(a, b) AS t from testData2) tmp").toDF(),
       testData2)
   }
+
+  test("Column aliases are ignored in callUDF while using struct()") {
```

Comment:
I checked and the test passes as @marmbrus said. Since this is old PR (3 months), I think this bug is already fixed in current codebase. So I will close this now.